app/controllers/items_controller.rb

```diff
@@ -85,11 +85,16 @@ def auto_complete
 	end
 
 	def add
-		items = params[:items][:ids].split(",").map do |id|
-			Item.find_by_id(id) || Item.create(name: id, price_cents: 0)
+    # Id is either an id of a known item or a name for a new one
+		params[:items][:ids].split(",").each do |id|
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.<br>Tab detected.